src/components/revaluation/ApartmentRevaluationModal.module.scss

```diff
@@ -0,0 +1,7 @@
+form {
```

Comment:
I don't think that this actually targets any `<form>` components as it should be referenced in the `.tsx` file as a `className`

Reply:
It does have an effect, but good catch I'll change it to a class reference so it doesn't cause any unintended side effects to other forms.